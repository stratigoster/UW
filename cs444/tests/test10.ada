PACKAGE foo IS
  default : INTEGER := 10/(3); -- this truncates :)
  s : STRING := "Batman";
  t : STRING := "Superman";
  b : BOOLEAN := s < t AND NOT (s = t) OR ELSE (default = default);
END foofoo;
--  ^^^^^^
-- expected `foo' but found `foofoo'
