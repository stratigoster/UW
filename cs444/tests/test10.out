***ERROR***:1:9:	Package name declaration and closure are not the same

------
Compilation error
------
